### ISSUE TITLE
Icons do not have tooltips / alt-text

### ISSUE THREAD

**Issue by jhesketh** [rand256/valetudo#300]
**Describe the bug**
Some of the icons (specifically on the map page) are not clear what they do. For example, does the map marker do a zoned cleanup, or locate the robot. Additionally sometimes the Play button means resume the clean and sometimes it means start a new clean.

It would be helpful if there were title/tooltips/alt-text on all of the buttons to make it clearer what action will be taken.

**How to Reproduce**
1. Go to map page
2. Hover over icon

**Expected behavior**
When hovering over any button, a textual description is provided.

**Vacuum Model:**
NA

**Valetudo Version:**
v0.9.6

**User-Agent**
Firefox

**Comment by rand256** [comment]
Yes, probably these texts should be added at some point. The initial reason for them being missing is that the web interface is supposed to be used mostly from a smartphone, which has no way to display tooltips upon cursor hovering buttons due to, well, the lack of any cursor on a touch screen.

**Comment by jhesketh** [comment]
That's fair. It is difficult to display on mobile. A long press could show what the action does though perhaps?

Alternatively you could look at a "sheet" (https://material.io/components/sheets-bottom) or an extended floating action button (https://material.io/components/buttons-floating-action-button#extended-fab).

Either way, just an alt-text for desktop users would be helpful :-).

**Comment by pidator** [comment]
> A long press could show what the action does though perhaps?

Some of the buttons on map tab do have already an additional action while long pressing them ;)

**Comment by avierck** [comment]
From a UI/UX perspective I would actually prefer it _a lot_ if the icons on the map would get a similar treatment to those on the bottom navigation bar: just give them localised text labels.
